hipack

```diff
@@ -62,35 +62,38 @@ formats = {
 }
 
 
-import optparse
-parser = optparse.OptionParser("[-f format] [-t format] < input > output")
-parser.add_option("-f", "--from", dest="from_format", default="json",
-        help="Read input in the specified FORMAT [default: %default]",
-        metavar="FORMAT")
-parser.add_option("-t", "--to", dest="to_format", default="hipack",
-        help="Write output in the specified FORMAT [default: %default]",
-        metavar="FORMAT")
-parser.add_option("--formats", default=False, action="store_true",
-        help="Print list of supported formats and exit")
-parser.add_option("--hipack-module-version", default=False,
-        action="store_true", dest="hipack_module_version",
+import argparse, sys
```

Comment:
Nope, cause is using ArgumentParser() instead of OptionParser(). Is a different module that supports argparse.FileType() that is much better to read and write files ;)